### DISCUSSION TITLE
`client` repo > naming for `handleUpdate` function

### DISCUSSION THREAD

**Discussion by hens-virtuella-varld** [data-interaction-fed24/server#11] [Q&A] [answered]
Does `handleUpdate` function add (create) a new todo? 
Would it be better to rename it to `handleCreate`? (since update is to edit an old todo.)

**Comment by Liam-hi** [comment, marked as answer]
I think I already answered what the handleUpdate do but I'm know sure if I answered your question correctly. Yes! I think is a good idea to change the names in order to improve the readability and to make the code more understandable.
